packages/beeq-tailwindcss/src/theme/default/light.ts

```diff
@@ -2,193 +2,194 @@ export const DefaultLightTheme = {
   /* -------------------------------------------------------------------------- */
   /*                              Background colors                             */
   /* -------------------------------------------------------------------------- */
-  'bq-background--primary': 'var(--bq-neutral-white)',
-  'bq-background--secondary': 'var(--bq-grey-100)',
-  'bq-background--tertiary': 'var(--bq-grey-200)',
-  'bq-background--alt': 'var(--bq-grey-300)',
-  'bq-background--inverse': 'var(--bq-grey-900)',
-  'bq-background--brand': 'var(--bq-iris-600)',
-  'bq-background--overlay': 'var(--bq-grey-900)', // with opacity: 60%
+  'bq-background--primary': 'var(--bq-white)',
+  'bq-background--secondary': 'var(--bq-neutral-100)',
+  'bq-background--tertiary': 'var(--bq-neutral-200)',
+  'bq-background--alt': 'var(--bq-neutral-300)',
+  'bq-background--inverse': 'var(--bq-neutral-900)',
+  'bq-background--brand': 'var(--bq-brand)',
+  'bq-background--overlay': 'var(--bq-neutral-900)', // with opacity: 60%
   /* -------------------------------------------------------------------------- */
   /*                                 Data colors                                */
   /* -------------------------------------------------------------------------- */
-  'bq-data--blue': 'var(--bq-blue-600)',
-  'bq-data--brand': 'var(--bq-iris-600)',
-  'bq-data--corai': 'var(--bq-corai-600)',
-  'bq-data--cyan': 'var(--bq-cyan-600)',
-  'bq-data--green': 'var(--bq-green-600)',
-  'bq-data--grey': 'var(--bq-grey-600)',
-  'bq-data--inverse': 'var(--bq-grey-950)',
+  // probably will be removed from here and added to the root file
+  'bq-data--blue': 'var(--bq-data-09)',
+  'bq-data--brand': 'var(--bq-brand)',
+  'bq-data--corai': 'var(--bq-corai-600)', // not found in root
+  'bq-data--cyan': 'var(--bq-data-04)',
+  'bq-data--green': 'var(--bq-green-600)', // not found in root
+  'bq-data--grey': 'var(--bq-data-10)',
+  'bq-data--inverse': 'var(--bq-neutral-950)',
   'bq-data--lime': 'var(--bq-lime-600)',
-  'bq-data--magenta': 'var(--bq-magenta-600)',
-  'bq-data--orange': 'var(--bq-orange-600)',
-  'bq-data--purple': 'var(--bq-purple-600)',
-  'bq-data--red': 'var(--bq-red-600)',
-  'bq-data--sky': 'var(--bq-sky-600)',
-  'bq-data--teal': 'var(--bq-teal-600)',
-  'bq-data--yellow': 'var(--bq-yellow-600)',
+  'bq-data--magenta': 'var(--bq-data-03)',
+  'bq-data--orange': 'var(--bq-data-06)',
+  'bq-data--purple': 'var(--bq-accent)',
+  'bq-data--red': 'var(--bq-danger)',
+  'bq-data--sky': 'var(--bq-sky-600)', // not found in root
+  'bq-data--teal': 'var(--bq-data-05)',
+  'bq-data--yellow': 'var(--bq-data-07)',
```

Comment:
I'm confused here, if we are going to remove the data group from the light/dark files, why bother updating the value? 😉 The new ones will be those added in the theme root.ts file, right?